tests/probes_test.go

```diff
@@ -102,18 +92,20 @@ var _ = Describe("[ref_id:1182]Probes", func() {
 			table.Entry("[test_id:1219][posneg:negative]with working HTTP probe and no running server", httpProbe),
 		)
 
-		table.DescribeTable("should succeed when probing an IPv6 IP on dual stack clusters", func(readinessProbe *v12.Probe) {
-			libnet.SkipWhenNotDualStackCluster(virtClient)
+		table.DescribeTable("should succeed when probing both IPv6 and IPv4 IP on dual stack clusters", func(readinessProbe *v12.Probe, IPFamily v1.IPFamily) {
```

Comment:
You should add this on lines 62-78 instead, and remove this part (so we keep the positive tests before the negative ones).

Reply:
Done. Removed the whole create-server-on-vmi part.